Gemfile

```diff
@@ -4,7 +4,6 @@ source "https://rubygems.org"
 gemspec
 
 gem "mixlib-shellout", "< 3.2.3"
```

Comment:
Is there a reason we're pinning this to an older release?

Reply:
No?

Reply:
Killing that in the GEMFILE and the gemspec file.